contributing/code/bc.rst

```diff
@@ -258,6 +258,14 @@ Make public or protected                                                  Yes
 Remove private property                                                   Yes
 **Constructors**
 Add constructor without mandatory arguments                               Yes             :ref:`[1] <note-1>`
+:ref:`Add argument without a default value <add-argument-public-method>`  No
+Add argument with a default value                                         Yes
+Remove argument                                                           No              :ref:`[3] <note-3>`
+Add default value to an argument                                          Yes
+Remove default value of an argument                                       No
+Add type hint to an argument                                              No
+Remove type hint of an argument                                           No
```

Comment:
this is actually allowed on a constructor, as it is a type of changed allowed for argument types (allowing anything is a super type of any existing type)

Reply:
That's right, updated